lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -3094,18 +3094,16 @@ balanceTransaction
 
                 mkRecentEra :: Handler (WriteTx.RecentEra era)
                 mkRecentEra = case Cardano.cardanoEra @era of
+                    Cardano.ConwayEra -> pure WriteTx.RecentEraConway
                     Cardano.BabbageEra -> pure WriteTx.RecentEraBabbage
-                    Cardano.AlonzoEra -> pure WriteTx.RecentEraAlonzo
                     _ -> liftHandler $ throwE $ Write.ErrOldEraNotSupported era
 
                 mkLedgerUTxO
                     :: [ApiExternalInput n]
                     -> Handler (WriteTx.UTxO (ShelleyLedgerEra era))
                 mkLedgerUTxO ins = do
                     recentEra <- mkRecentEra
-                    liftHandler
-                        . ExceptT
-                        . pure
+                    return
```

Comment:
I usually use `pure` in this cases at its less powerful but sufficient, as per https://www.lihaoyi.com/post/StrategicScalaStylePrincipleofLeastPower.html#philosophy-principle-of-least-power